noxfile.py

```diff
@@ -48,3 +51,58 @@ def run_integration_test_list(session: nox.Session):
             if "gpu" not in t.name
         ]
     print(json.dumps(tests))
+
+
+@nox.session(name="cleanup-docker-hub", python=False)
+def cleanup_docker_hub(session: nox.Session):
+    """
+    Removes docker hub tags older than `min_age_in_days` days.
+    """
+    parser = ArgumentParser(
+        usage=f"nox -s {session.name} -- --docker-repository <repository> --docker-username <username> --docker-password <password> --min-age-days <min_age_in_days>"
+    )
+    parser.add_argument(
+        "--docker-repository",
+        type=str,
+        required=True,
+        help="Docker Repository name",
+    )
+    parser.add_argument(
+        "--docker-username",
+        type=str,
+        required=True,
+        help="Docker username",
+    )
+    parser.add_argument(
+        "--docker-password",
+        type=str,
+        required=True,
+        help="Docker password/PAT",
+    )
+    parser.add_argument(
+        "--min-age-in-days",
+        type=int,
+        required=True,
+        help="Minimum age in days of tags which will be removed",
+    )
+    parser.add_argument(
+        "--max-number-pages",
+        type=int,
+        default=100,
+        required=False,
+        help=cleandoc(
+            """Maximum number of pages.
+        The page size is fixed at 100 (maximum from Dockerhub RestAPI).
+        This means 100*MAX_NUMBER_PAGES will be deleted.
```

Comment:
```suggestion
        This means 100*MAX_NUMBER_PAGES tags can be deleted.
```

Reply:
As a user, I'd prefer not needing to multiply page size by number of pages. 
Could we think of a parameter like `max_tags` and let the implementation do the math?

Reply:
I replace '--max-number-pages' with '--max-number-tage'